client/common/client_cliprdr_file.c

```diff
@@ -152,6 +152,55 @@ struct cliprdr_file_context
 	BYTE client_data_hash[WINPR_SHA256_DIGEST_LENGTH];
 };
 
+struct escape_pair
+{
+	const char* what;
+	const char* escape;
+};
+
+static const struct escape_pair url_escape_codes[] = {
+	{ " ", "\%20" }, { "<", "\%3C" }, { ">", "\%3E" }, { "#", "\%23" },  { "%", "\%25" },
```

Comment:
I don't get why you need \ in`\%` ?

Reply:
@hardening true, not really required here.